ecl/AccuracyTestModified.ecl

```diff
@@ -0,0 +1,96 @@
+/*##############################################################################
+    
+    HPCC SYSTEMS software Copyright (C) 2022 HPCC Systems®.
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+       
+       http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+############################################################################## */
+
+#ONWARNING(6003, ignore);
```

Comment:
After the ML-495 will be merged (I tested, reviewed and approved that), this error won't appear again, please remove this. (BTW it has no affect, because the error comes from the compiler and not the runtime.)